dnf/cli/commands/install.py

```diff
@@ -127,3 +129,17 @@ def run(self):
             raise dnf.exceptions.PackagesNotAvailableError(
                 _("Unable to find a match"), pkg_spec=' '.join(errs),
                 packages=err_pkgs)
+
+    def check_arguments(self):
+        security_filters = [self.opts.advisory, self.opts.bugzilla,
+                            self.opts.cves, self.opts.severity]
+        specs = [self.opts.pkg_specs, self.opts.grp_specs, self.opts.filenames]
+        if any(security_filters):
+            if all(not spec for spec in specs):
+                self.opts.pkg_specs.append("*")
+            if self.base._update_security_filters:
```

Comment:
This 'if' will not work here well. The error message has to be raised in base.install() when there is something in query, but after security filter application query is empty. Also reporting an argument that doesn't have  security upgrade will be nice.